src/Network.ts

```diff
@@ -63,13 +60,13 @@ export default class Network {
         url = `http:${url.substring(6)}`
       }
       // eslint-disable-next-line prefer-rest-params
-      return apply.call(Network.ajaxIE8, null, arguments)
+      return Function.prototype.apply.call(this.ajaxIE8.bind(this), null, arguments)
```

Comment:
nit: why not `this.ajaxIE8.apply(this, arguments)`?

Reply:
Thanks, updated as suggested!